pheweb/load/download_rsids.py

```diff
@@ -15,7 +15,10 @@ def run(argv):
 
             # dbSNP downloads are described at <https://www.ncbi.nlm.nih.gov/variation/docs/human_variation_vcf/>
             # This file includes chr-pos-ref-alt-rsid and 4X a bunch of useless columns:
-            dbsnp_url = 'ftp://ftp.ncbi.nlm.nih.gov/snp/organisms/human_9606_b150_GRCh38p7/VCF/00-All.vcf.gz'
+            if genome_build == '37':
```

Comment:
check for allowed values in genoe_build and error out if wrong. Catches typo style configuration errors early.